### DISCUSSION TITLE
请求添加茶杯头

### DISCUSSION THREAD

**Discussion by liushengqi000** [Gamuxorg/bbs#236] [投递和请求游戏]
游戏名称：茶杯头
游戏名称(英文)：CUPHEAD
下载地址 :（如果有的话）：

**Comment by liushengqi000** [comment]
刚注意没有原生linux....OTZ，就不勉强了

**Comment by liushengqi000** [comment]
噗，你们是圣诞老人嘛。居然真的做出来了，stream上都没看到linux支持的说OTZ。太厉害了

**Comment by hyh19962008** [comment]
哈哈哈，感谢持续关注本站

Best

________________________________
